CHANGELOG.md

```diff
@@ -5,6 +5,9 @@ Changelog
 
 ## Enhancements
 
+* Use new Apple API to get free memory, if available.
```

Comment:
Can we just say "Now using os_proc_available_memory" instead of "new Apple API"?